src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,320 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 204-10-18 06:00:00
+layout: post
+title: "Call for Volunteers: DjangoCon US 2025"
+---
+
+As our program chair, Drew Winstel, mentioned in his opening remarks today, we would love to have more volunteers for DjangoCon US 2025. Expect to hear details on dates and venues within the next month or so.
+
+We have some [documentation about the various roles](https://djangocon.github.io/djangocon-us-docs/), but here's a quick overview of the teams at DjangoCon US.
+
+The DEFNA board will not leave you stranded; we have a lot of documentation to help you on your way, including rough checklists of tasks needed for each team.
+
+Ideally, all committees will have a chair and co-chair to allow for one or the other to take vacations, handle day job duties, and so forth. This list is by no means comprehensive, but it should hopefully give you an idea of what each committee is responsible for.
+
+## Conference Leadership
+
+The conference chair and co-chair lead the conference, including coordinating between teams and making executive decisions (including approaching the DEFNA board when applicable). Expect to be in meetings with the venue and online conference provider, weekly organizer check-ins, monthly reports to the DEFNA board, and lots of questions on Slack. You'll also lead the volunteer kickoff meetings in early 2025 (ideally mid-to-late January) and recruit organizers as you see fit. You'll need to recommend an online conference provider to the DEFNA board (might be part of a sponsorship deal). This year we used [Venueless](https://venueless.org/), though that's not a requirement.
+
+At the conference, you'll be making speeches at the start of each day and the end-of-conference wrap-up in addition to coordinating setup, teardown, and everything in between.
+
+These two roles have the highest time commitment and the most visibility. If you want to help people get things done and get up on stage, these are the roles for you.
+
+## Program Committee
+
+Size: 2-4
+
+The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be.
```

Comment:
```suggestion
The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be. There is both a high degree of influence over the tutorials and talks and a high time commitment when you're a chair of the program committee.
```

Reply:
@drewbrew do you want this called out explicitly? I tried to make it not sound like an awful gig.

Reply:
@drewbrew do you want to change this before we post it later this week?